test/module/irohad/ametsuchi/flat_file_test.cpp

```diff
@@ -101,3 +101,29 @@ TEST_F(BlStore_Test, BlockStoreWhenAbsentFolder) {
   }
   rmdir(target_path.c_str());
 }
+
+/**
+ * @given non-empty folder from previous block store
+ * @when new block storage is initialized
+ * @then new block storage has all blocks from the folder
+ */
+TEST_F(BlStore_Test, BlockStoreInitializationFromNonemptyFolder){
+  auto bl_store1 = FlatFile::create(block_store_path);
+  ASSERT_TRUE(bl_store1);
+
+  // Add two blocks to storage
+  std::vector<uint8_t > block1(1000, 5);
+  auto id1 = 1u;
+  bl_store1->add(id1, block1);
```

Comment:
as you use previous vars once you can inline call:
`b1_store1->add(1u, std::vector<uint8_t>(1000, 5));`
same for the following